### ISSUE TITLE
Data views: Add "Password Protected" to Page and Post Status Options for bulk changes

### ISSUE THREAD

**Issue by cuemarie** [WordPress/gutenberg#49012]
## What problem does this address?

User would like to see the Password Protected option added to the possible statuses given in Bulk Edit for Pages and Posts.

Suggestion came from a user who wants to have all pages on their site Password Protected except the home page, but without having to add the password to each individual page manually.

Originally reported here: https://github.com/Automattic/wp-calypso/issues/74356

**Comment by kathrynwp** [comment]
Hi @cuemarie - thanks for submitting this issue. Could you please confirm whether you're referring to the bulk-edit screen here:

<img width="1488" alt="Pages_‹_Test_Site_ABC_—_WordPress" src="https://user-images.githubusercontent.com/7662449/224816618-51c49243-7402-4105-a20e-1a047f101c4b.png">

If so, I think this would be a Core change, rather than a Gutenberg one, and a ticket should be opened in [Core Trac](https://core.trac.wordpress.org/).

**Comment by Mamaduka** [comment]
The editor doesn't handle the bulk edit actions UI. The request should be submitted in Core Trac.

**Comment by annezazu** [comment]
Reopening this considering data views is bringing bulk edit actions into Pages for now within the Site Editor cc @jameskoster for consideration as part of quick actions: https://github.com/WordPress/gutenberg/issues/55628

**Comment by jameskoster** [comment]
Yup this should definitely be possible, ideally via quick edit (https://github.com/WordPress/gutenberg/issues/55101) so that it can be applied in bulk as requested in the OP. The design should be aligned with the full screen editor which currently groups visibility options with status:

<img width="655" alt="Screenshot 2024-09-10 at 10 15 08" src="https://github.com/user-attachments/assets/a3ca7bb4-21a9-4323-a4c0-352b92faeac8">

Adding it to quick actions could make sense too. It may be worth opening a separate issue for that one.

**Comment by youknowriad** [comment]
You see the faint button on the screenshot, this is already possible in the quick edit in "pages". I would say that the fact that the button is "empty" is a small bug though.

<img width="341" height="211" alt="Image" src="https://github.com/user-attachments/assets/51127445-a89b-4e78-8adf-67bb893d8148" />
